public/gameInfo.php

```diff
@@ -1031,6 +1047,17 @@ function submitSetRequest(user, gameID) {
 
                 // Display the claims links if not an event game
                 if (!$isEventGame) {
+                    if ($permissions >= Permissions::Developer) {
+                        $gameMetaBindings['developListType'] = UserGameListType::Develop;
+                        echo Blade::render('
+                            <x-game.add-to-list
+                                :gameId="$gameID"
+                                :type="$developListType"
```

Comment:
It's hard not to notice the new button having a different letter spacing, font size, and height from the other buttons in the devbox. Visually, the difference is striking enough to mistake the new button as being a "primary action" (the new button is nearly 2x the height of the others).

Recommend when the `type` prop is `UserGameListType::Develop`, the button styles change to match the other buttons in the devbox.

![Screenshot 2023-09-11 at 6 57 20 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/d7255793-ff24-483e-9189-c86289462509)

Reply:
I can fix the font size by removing the `text-sm` from the span block, but everything I try to set for the image sizes cause the image to disappear.

https://github.com/RetroAchievements/RAWeb/blob/4d1acc47288928caadae2c790cdb00dcb23d9b5c/resources/views/platform/components/game/add-to-list.blade.php#L30-L36

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/b14e1b2a-55f0-448a-911d-20727342224b)

I don't think that the logic need be conditional. A slightly smaller button in the header would be fine.

Reply:
It looks like something in the SVG for the icons is causing the issue. This is where I ended up:

```php
<button 
    id='{{ $type }}-list-button'
    class='btn'
    type='button'
    title='{{ $buttonTooltip }}'
    onClick="toggleListItem('{{ $type }}')"
>
    <div class="flex items-center gap-x-1">
        <div
            id='add-to-{{ $type }}-list'
            class="{{ $addVisibility }} {{ $type === 'play' ? 'w-[18px] h-[18px]' : '' }}"
        >
            @if ($type === 'play')
                <x-icon.add-to-list />
            @else
                <x-fas-plus class="-mt-0.5 w-[12px] h-[12px]" />
            @endif
        </div>

        <div
            id='remove-from-{{ $type }}-list'
            class="{{ $removeVisibility }} {{ $type === 'play' ? 'w-[18px] h-[18px]' : '' }}"
        >
            @if ($type === 'play')
                <x-icon.remove-from-list />
            @else
                <x-fas-check class="-mt-0.5 w-[12px] h-[12px]" />
            @endif
        </div>

        <span>
            {{ __('user-game-list.' . $type . '.name') }}
        </span>
    </div>
</button>
```

![Screenshot 2023-09-12 at 7 57 46 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/903c59ea-cacc-4c38-b12b-9a5a71a98a9f)

Reply:
I think the non-circle icons look find for the Want to Play button and removed the conditional logic. Thanks for your help.